From 422abaa114dd8641e8ef45067627290970a12a3a Mon Sep 17 00:00:00 2001
From: czou <cao.zou@windriver.com>
Date: Tue, 8 Sep 2015 17:36:59 +0800
Subject: [PATCH] dts: audio: twl4030: add twl4030 dts for omap3evm

now twl4030 audio driver supports the dtb, so add the omap3evm
audio dtb data.

Signed-off-by: czou <cao.zou@windriver.com>
---
 arch/arm/boot/dts/omap3-evm.dts | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm/boot/dts/omap3-evm.dts b/arch/arm/boot/dts/omap3-evm.dts
index 11e4f29..24507a6 100644
--- a/arch/arm/boot/dts/omap3-evm.dts
+++ b/arch/arm/boot/dts/omap3-evm.dts
@@ -34,6 +34,14 @@
 		};
 	};
 
+	sound {
+		compatible = "ti,omap-twl4030";
+		ti,model = "omap3evm";
+
+		ti,mcbsp = <&mcbsp2>;
+		ti,codec = <&twl_audio>;
+	};
+
 	/* HS USB Host PHY on PORT 2 */
 	hsusb2_phy: hsusb2_phy {
 		compatible = "usb-nop-xceiv";
@@ -81,6 +89,16 @@
 	};
 };
 
+&i2c1 {
+	twl: twl@48 {
+		twl_audio: audio {
+			compatible = "ti,twl4030-audio";
+			codec {
+			};
+		};
+	};
+};
+
 &twl_gpio {
 	ti,usb-vbus = <1>;
 };
-- 
1.9.1

